RuntimeUnityEditor/Windows/Profiler/ProfilerWindow.cs

```diff
@@ -307,10 +321,25 @@ private IEnumerator FrameEndCo()
                     }
                 }
 
-                if (_needResort)
+                if (_needResort || _aggregation)
```

Comment:
This appears to be a part of the issue, it shouldn't be resorting on every frame.

Reply:
Required to update aggregated values.
Aggregated values are not updated by Postfix.